media/js/src/simulations/simulation2/components/simulationTwo.jsx

```diff
@@ -75,6 +90,9 @@ export const SimulationTwo = () => {
     const handleStartOver = () => {
         setData();
         setControls({});
+        for (const topic in isComplete) {
+            updateAnswer(submissionId, takeaways2[topic].q_id, {active: false});
```

Comment:
We should make a distinction between what updating an answer is and what a delete is. If the goal is to mark an answer inactive then I think we should rename the view/route/etc.. to 'delete answer'. I think that makes things clearer.

Reply:
Delete answer already exists, but it removes the object from the database. I thought we wanted to store past attempts. Wasn't that the point of the active column?

Reply:
DeleteAnswerView just marks it as inactive

Reply:
Ugh. I misread it. Sorry about that. Could have spared myself some time building the other view.